mappers/quality-label.js

```diff
@@ -19,13 +19,13 @@ function mapAccessibilityLabel(recordId, record) {
 
       return { uri, statements };
     } else if (!Object.keys(accessibilityLabels).includes(record['accessibility_label'])) {
-      console.error(`Cannot map accessibility label value '${record['accessibility_label']}' for record ${recordId}`);
+      errorLogger('accessibility label', record['accessibility_label'], recordId);
```

Comment:
Pass the same field name (`accessibility_label`) as used to lookup the value as first argument. I know it was different before, but old code is wrong.